templates/base.html.template

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+
+    <!--
+      This was likely rendered from the Privly-application templating system
+      See: https://github.com/privly/privly-applications/pull/19
+     -->
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    
+    <link href="../shared/css/common.css" media="screen" rel="stylesheet" type="text/css"/>
+    
+    <script type="text/javascript" src="../shared/javascripts/meta_loader.js"></script>
+    <script type="text/javascript" src="../shared/javascripts/parameters.js"></script>
+    <script type="text/javascript" src="../shared/javascripts/network_service.js"></script>
+    <script type="text/javascript" src="../shared/javascripts/local_storage.js"></script>
+
+    {% block head %}{% endblock %}
+  </head>
+  {% block body %}{% endblock %}
```

Comment:
The head opening/closing tags are in the base template, but the body tags are not. I think it may be more flexible to include the body tag here since it will allow for the base template to include elements in the body without changing the extending template. Hard to say. I think I need to spend time playing with this.